tests/zfs-tests/tests/functional/cli_root/zpool_clear/zpool_clear_001_pos.ksh

```diff
@@ -192,6 +192,7 @@ function do_testing #<clear type> <vdevs>
 		    log_fail "'zpool clear' fails to clear error for pool $TESTPOOL1."
 	fi
 
+	log_must zpool wait -t scrub $TESTPOOL1
```

Comment:
Could you explain why this additional wait is needed.  Is the `zpool scrub -w` on line 179 occasionally not waiting until the scrub completes?   Or is there some other case here?

Reply:
at lines 184 and  190 you can see `log_must zpool clear $TESTPOOL1`
some time it start scrub and we try destroy pool and i see message about pool is busy

Reply:
Thanks for the clarification, though I think it may just raise more questions.
1) `zpool clear` shouldn't trigger a new scrub
2) `zpool destroy` should allow you to destroy a pool even when a scrub is in progress.
Do you know if this behavior is intentionally different on dilos?  I wouldn't have expected a `pool is busy` error due to a running scrub.